setup_wordpress.sh

```diff
@@ -4,3 +4,68 @@ set -e
 
 # Your code goes below ...
 echo 'This script should install and setup Wordpress'
+
+echo '---------- Update & Upgrade APT ----------'
+sudo apt-get update -y
+sudo apt-get upgrade -y
```

Comment:
why do we upgrade?

Reply:
Update = updates the list of available packages but doesnt install it.
Upgrade = installs these packages from the last updated version

Reply:
but why do you need latest version of all installed packages?